node/sc/remote_backend.go

```diff
@@ -163,14 +163,31 @@ func (rb *RemoteBackend) TransactionReceipt(ctx context.Context, txHash common.H
 	return r, err
 }
 
+func (rb *RemoteBackend) TransactionReceiptRpcOutput(ctx context.Context, txHash common.Hash) (map[string]interface{}, error) {
```

Comment:
how about this?
`func (rb *RemoteBackend) TransactionReceiptRpcOutput(ctx context.Context, txHash common.Hash) (r map[string]interface{}, error) {`
~var r map[string]interface{}~